version/derive/src/lib.rs

```diff
@@ -2,7 +2,9 @@
     clippy::module_name_repetitions,
     missing_docs,
     clippy::shadow_reuse,
-    clippy::str_to_string
+    clippy::str_to_string,
+    clippy::arithmetic,
+    clippy::std_instead_of_core,
```

Comment:
I think we should disable this lint. I don't see what it brings us. If we use `no_std` then the crate will not compile and this is covered in CI

Reply:
If I add that to `lints.toml` it will stop `cargo lints` from working. The lint is present in the compatible nightly, but not in  the latest stable.